bundles/org.openhab.binding.liquidcheck/src/main/resources/OH-INF/i18n/liquidcheck_en.properties

```diff
@@ -0,0 +1,33 @@
+# binding
```

Comment:
Please rename this file liquidcheck.properties.
After you updated your XML files, umdate it automatically by running "mvn i18n:generate-default-translations".